keras/layers/containers.py

```diff
@@ -139,6 +139,22 @@ def __init__(self):
         self.output_config = []  # dicts
         self.node_config = []  # dicts
 
+    def _apply(self, ins, train=True):         
```

Comment:
Suggestions (that need to be discussed):
* Could we get something like that in a `layer`? Container is a layer, could it inherit from there instead?
* Instead of `_apply`, this could be `__call__`. So we could use the container simply as `model(ins)` instead of `model._apply(ins)`. This is how it is done in TensorFlow for instance.